roles/ssh_hardening/vars/Fedora_37.yml

```diff
@@ -0,0 +1,24 @@
+---
+sshd_path: /usr/sbin/sshd
+ssh_host_keys_dir: /etc/ssh
+sshd_service_name: sshd
+ssh_owner: root
+ssh_group: root
+ssh_host_keys_owner: root
+ssh_host_keys_group: ssh_keys
```

Comment:
This file only exists due to this line - see https://github.com/dev-sec/ansible-collection-hardening/pull/677 for more context.